config/rbac/role.yaml

```diff
@@ -5,6 +5,12 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - appstudio.redhat.com
+  resources:
+  - applicationsnapshots
```

Comment:
Is `applicationsnapshots` a custom resource? Maybe not the best place to ask about this :D

Reply:
It is. You can find it under the `api` directory.